test/runtests.jl

```diff
@@ -61,6 +61,11 @@ include(joinpath(examples_path, "kinsol_mkinTest.jl"))
 @test abs(minimum(residual)) < 1e-5
 end
 
+println("== handle tests")
+let
+@testset "Handle Tests" begin include("handle_tests.jl") end
```

Comment:
I think here `@testset` is not necessary, but printing the file being executed would be nice.
I don't know what is the most "idiomatic" way to organize Julia package tests, but you can look into e.g. [DataTables.jl runtests.jl](https://github.com/JuliaData/DataTables.jl/blob/master/test/runtests.jl) for how it is done (a tiny infrastructure to run tests from multiple files).